.changeset/popular-countries-wave.md

```diff
@@ -0,0 +1,5 @@
+---
+"@apollo/utils.keyvaluecache": patch
+---
+
+Update lru-cache dependency to v8.x
```

Comment:
IIUC we use lru-cache's options type as our type so this actually may have user-visible effects... maybe describe in more detail or link to changelog?

Reply:
Yeah good callout. [476399f](https://github.com/apollographql/apollo-utils/pull/271/commits/476399f86022733b8e0485467b6dc1a7968b83f5)